src/main.rs

```diff
@@ -143,6 +143,12 @@ fn notmain() -> anyhow::Result<i32> {
     let max_backtrace_len = opts.max_backtrace_len;
     let shorten_paths = opts.shorten_paths;
     let elf_path = opts.elf.as_deref().unwrap();
+    if !elf_path.exists() {
+        return Err(anyhow!(
+            "Can't find ELF file at `{}`; are you sure you got the right path?",
```

Comment:
Error messages are usually lowercased:

```suggestion
            "can't find ELF file at `{}`; are you sure you got the right path?",
```

Other than that, seems good, feel free to r+